kasa/modules/motion.py

```diff
@@ -59,3 +59,16 @@ async def set_range(
             )
 
         return await self.call("set_trigger_sens", payload)
+
+    @property
+    def inactivity_timeout(self) -> int:
+        """Return inactivity timeout in milliseconds."""
+        return self.data["cold_time"]
+
+    async def set_inactivity_timeout(self, timeout: int):
+        """Set inactivity timeout in milliseconds.
+
+        Note, that you need to delete the default rule to avoid reverting this
+        back to 60 seconds after a period of time.
```

Comment:
It might be worth mentioning that the offending default rule is one of what TP-Link calls "Smart Control" rules, which are managed by the `smartlife.iot.sensor_trigger` module. The following command will probably delete the offending rule:
```
{"smartlife.iot.sensor_trigger": {"delete_weekday_routine": {"id": "0"}}}
```
Note, to really do it properly, you shouldn't assume that the `id` of the rule will be `"0"`. Instead, you should call `get_weekday_routine` and iterate over the returned `array` to find the `id` of each existing rule, which you can then pass in turn to `delete_weekday_routine`. On my KS200M, the one allowed Smart Control rule is always given `"id": "0"`, but that seems to be an anomaly, as all other rule sets in the Kasa API (that I've encountered at least) use random hex strings as rule identifiers.